pkg/slayers/path/empty/empty.go

```diff
@@ -55,6 +55,7 @@ func RegisterPath() {
 // bytes on the wire and is used for AS internal communication.
 type Path struct{}
 
+// @ requires low(len(r))
```

Comment:
In the future, if we want to prove that the resulting path is low, it may be worth start thinking about introducing hyper functions for lowness of the contents of slices. I would suggest making such hyper functions opaque, to preserve good perf.